internal/runners/config/set.go

```diff
@@ -45,11 +50,23 @@ func (s *Set) Run(params SetParams) error {
 		return locale.WrapError(err, "err_config_set_event", "Could not store config value, if this continues to happen please contact support.")
 	}
 
-	err = s.cfg.Set(params.Key.String(), value)
+	key := params.Key.String()
+
+	err = s.cfg.Set(key, value)
 	if err != nil {
 		return locale.WrapError(err, "err_config_set", fmt.Sprintf("Could not set value %s for key %s", params.Value, params.Key))
 	}
 
+	// Notify listeners that this key has changed.
+	configMediator.NotifyListeners(key)
+
+	// Notify state-svc that this key has changed.
+	if s.svcModel != nil {
+		if err := s.svcModel.ConfigChanged(context.Background(), key); err != nil {
```

Comment:
Please double check this wouldn't create a recursion issue if we set config values in the state-svc.

Reply:
I added the conditional, but this wouldn't recurse because it happens in `state config set`, which would not be called by `state-svc`. I suppose even if it could, it would result in two notifications, but nothing more.

Reply:
Ahh I missed that in the review, in my head this was in the config package.